modules/mod_dubbo/ngx_http_dubbo_module.c

```diff
@@ -649,6 +653,39 @@ ngx_http_dubbo_create_dubbo_request(ngx_http_request_t *r, ngx_connection_t *pc,
     
     dlcf = ngx_http_get_module_loc_conf(r, ngx_http_dubbo_module);
 
+    service_name = ngx_palloc(r->pool, sizeof(ngx_str_t));
+    if (service_name == NULL) {
+        return NGX_ERROR;
+    }
+
+    if (ngx_http_complex_value(r, &dlcf->service_name, service_name)
+        != NGX_OK)
+    {
```

Comment:
code style on '{'  and some other

Reply:
Look like Nginx also put the `{` into a new line in similar case?
https://github.com/nginx/nginx/blob/402f4cda0c389242a587eff7bb55959663504c8b/src/http/modules/ngx_http_gzip_static_module.c#L158

Reply:
got it